alf/trainers/policy_trainer.py

```diff
@@ -217,8 +217,12 @@ def initialize(self):
             not self._use_tf_functions)
         self._env = create_environment()
         common.set_global_env(self._env)
+        # Create an unwrapped env to expose subprocess gin confs which otherwise
+        # will be marked as "inoperative"
+        unwrapped_env = create_environment(force_unwrapped=True)
```

Comment:
num_parallel_environments=1

Reply:
No need to specify that because if num_parallel_environments > 1, then it's definitely wrapped. force_unwrapped=True will only create a single env